src/Files.App/Helpers/FilePropertiesHelpers.cs

```diff
@@ -118,5 +121,21 @@ void Navigate(Frame frame)
 				frame.Navigate(typeof(Properties), argument, new SuppressNavigationTransitionInfo());
 			}
 		}
+
+		private static string GetFilesLogoPath()
+		{
+			var appTilesPath = Path.Combine(Package.Current.InstalledLocation.Path, "Assets/AppTiles");
+
+			if (Directory.Exists(Path.Combine(appTilesPath, "Dev")))
+				return Path.Combine(appTilesPath, "Dev", "Logo.ico");
+
+			if (Directory.Exists(Path.Combine(appTilesPath, "Preview")))
+				return Path.Combine(appTilesPath, "Preview", "Logo.ico");
+
+			if (Directory.Exists(Path.Combine(appTilesPath, "Release")))
+				return Path.Combine(appTilesPath, "Release", "Logo.ico");
```

Comment:
Following the comment above: 

```suggestion
                        var appTilesPreviewPath = Path.Combine(appTilesPath, "Preview");
			if (Directory.Exists(appTilesPreviewPath))
				return Path.Combine(appTilesPreviewPath, "Logo.ico");

                        var appTilesReleasePath = Path.Combine(appTilesPath, "Release");
			if (Directory.Exists(appTilesReleasePath))
				return Path.Combine(appTilesReleasePath,"Logo.ico");
```

Reply:
I modified GetFilesLogoPath with Linq and a sub function.